routing.asciidoc

```diff
@@ -35,126 +35,196 @@ We will discuss how the _structure_ (edges, fees, time-locks, etc) of the route
+Alice might also might wish to minimize the total number of channels she has open.
+Instead, Alice can repurpose her existing open channels to send a tip to Gloria, _without_ the need to open a channel directly with Gloria.
+This is possible, as long as there exists some path of channels from Alice to Gloria with sufficient capacity to route the tip.
+
+From previous chapters, we know Alice has an open channel with Bob, the coffee shop owner.
+Bob, in turn, has an open channel with the software developer Wei who helps him with technical issues of the point of sale system he uses in his coffee shop.
+Wei is the owner of a large software company which develops the game that Gloria plays, and they already have an open channel which Gloria uses to pay for the game's license and in game items.
+
+If we draw out this series of payment channels, it's possible to manually trace a _path_ from Alice to Gloria that uses Bob and Wei as intermediary routing nodes.
 Alice can then craft a _route_ from this outlined path, and use it to send a tip of a few thousand satoshis to Gloria, with the payment being _forwarded_ by Bob and Wei.
+Essentially, Alice will pay Bob, who will pay Wei, who will pay Gloria.
+And no direct channel from Alice to Gloria is required.
 
 [[routing-network]]
 .The network of payment channels of our friends can be seen here:
 image:images/routing-network.png[]
 
-The main challenge is to do this in a way that prevents Bob and Wei from stealing the money that Alice wants to be delivered to Gloria.
-To understand how the Lightning Network protects the payment packages that are being routed through the network we compare the situation of indirect payments with physical payments with gold coins in the offline world.
-Let us assume Alice wanted to give 10 gold coins to Gloria and decides to ask Bob and Wei for help.
-How could Alice make sure that Bob and Alice does not run with the gold coins after receiving them?
-In the physical world, traditional legal contracts contracts may be used to safely transmit this cascade of payments.
+The main challenge is to do this in a way that prevents Bob and Wei from stealing the money that Alice wants delivered to Gloria.
+To understand how the Lightning Network protects the payment while being routed, we can compare to an example of routing physical payments with golden coins in the real world.
+
+Assume Alice wants to give 10 gold coins to Gloria, but does not have direct access to Gloria.
+However, Alice knows Bob, who knows Wei, who knows Gloria and so she decides to ask Bob and Wei for help.
+She can pay Bob to pay Wei to pay Gloria, but how does she make sure that Bob or Wei don't run off with the coins after receiving them?
+In the physical world contracts could be used for safely carrying out a series of payments.
 
 Alice could negotiate a contract with Bob which reads:
 
-_I (Alice) will give you (Bob) 10 golden coins if you pass them on to Wei_
+[alice-gloria-routing-1]
```

Comment:
Not entirely sure if this is the best way to format these "contracts". I wanted to stand out from the regular text as essentially these "contracts" are a proxy for actual code being used on the network.